test/e2e/specs/onboarding/signup__with-theme-premium.ts

```diff
@@ -123,7 +123,7 @@ describe( 'Lifecyle: Premium theme signup, onboard, launch and cancel subscripti
 			await page.getByText( 'Customize this design' ).waitFor();
 		} );
 
-		it( 'Checks the active theme', async function () {
+		it.skip( 'Checks the active theme', async function () {
```

Comment:
The API that `getActiveTheme` uses in this test does not return a tiers key, so lets not add a taxonomy check, at least not now.

Checking that we got what we asked for should be enough.